config/environments/development.rb

```diff
@@ -28,7 +28,6 @@
     config.action_controller.perform_caching = true
     config.action_controller.enable_fragment_cache_logging = true
 
-    config.cache_store = :memory_store
```

Comment:
@newstler I'm not sure we want to remove this. I think the point of this block is that you should be able to test a caching setup locally by having a file located at `tmp/caching-dev.txt`. If you don't want to test caching you'd just remove that file. Does that sound right to you?

Reply:
Definitely that did it into PR occasionally, sure this line should stay unchanged.